File: ProgrammersLifeiOS/ProgrammersLifeiOS/Label.m
//
//  Label.m
//  ProgrammersLifeiOS
//
//  Created by Zackary T Misso on 7/10/15.
//  Copyright © 2015 Zackary Misso. All rights reserved.
//

#import "Label.h"

@implementation Label

-(id)init{
    if(self=[super init]){
        text = @"";
    }
    return self;
}

-(void)draw{
    // to be implemented
}

// getter methods
-(NSString*)getText { return text; }

// setter methods
-(void)setText:(NSString*)value { text = value; }

@end

File: ProgrammersLifeDemo/src/ui/Label.java
// This is the Game Tester's version of ProgrammersLife
// Author :: Zackary Misso / FiralStudios
// Version :: 0.0.1

package ui;
import java.awt.Graphics2D;
import gameState.GameView;

public class Label extends GameView {
	private String label;
	
	public Label(){
		super();
	}
	
	public void update(){
		// to be implemented
	}
	
	public void draw(Graphics2D g){
		// to be implemented
	}
	
	// getter methods
	public String getLabel(){return label;}
	
	// setter methods
	public void setLabel(String param){label = param;}
}
